src/types/Quote.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @module TypeInterfaces/quote
+ */
+/**
+ * Dummy comment needed for correct doc display, do not remove.
+ */
+
+import ICType from './CType'
+
+export interface ICostBreakdown {
+  tax: number
+  net: number
+  gross: number
+}
+export default interface IQuote {
+  attesterAddress: string
+  cTypeHash: ICType['hash']
+  cost: ICostBreakdown
+  currency: string
+  quoteTimeframe: Date
+  termsAndConditions: string
+  version: string
+}
+export interface IQuoteAttesterSigned extends IQuote {
+  quoteHash: string
+  attesterSignature: string
+}
+
+export interface IQuoteAgreement extends IQuoteAttesterSigned {
```

Comment:
Can the claimer only sign, when the attester signed it beforehand?

Reply:
Yes, we want that. 

We have it in stages matching the process of when the entity has the data.

Reply:
So we can retrace that it has been done correctly. 

If it switches it around we have to test both possibilities.